File: HB/pysrc/프로그래머스/레벨0/Day17/OX퀴즈.py
def solution(quiz):
    answer = []
    for i in range(len(quiz)):
        quiz_num=quiz[i].split(' ')
        a=0
        if quiz_num[1] =='+':
            a=int(quiz_num[0])+int(quiz_num[2])
        else:
            a=int(quiz_num[0])-int(quiz_num[2])
        if a==int(quiz_num[4]):
            answer.append('O')
        else:
            answer.append('X')
    return answer

File: SB/src/프로그래머스/레벨0/Day21/OX퀴즈.java
package 프로그래머스.레벨0.Day21;

public class OX퀴즈 {
    class Solution {
        public String[] solution(String[] quiz) {
            for (int i = 0; i < quiz.length; i++) {
                String sArr[] = quiz[i].split(" ");
                int result = Integer.parseInt(sArr[0]) + ((sArr[1].equals("+") ? +1 : -1) * Integer.parseInt(sArr[2]));
                quiz[i] = result == Integer.parseInt(sArr[4]) ? "O" : "X";
            }

            return quiz;
        }
    }
}
